transport/http_client.go

```diff
@@ -0,0 +1,19 @@
+package transport
+
+import (
+	"net/http"
+
+	"log"
+	"time"
+
+	"github.com/foundriesio/fioconfig/sotatoml"
+)
+
+func CreateClient(cfg *sotatoml.AppConfig) (*http.Client) {
+	tlsCfg, _, err := GetTlsConfig(cfg)
+	if err != nil {
+		log.Fatal(err)
+	}
+	transport := &http.Transport{TLSClientConfig: tlsCfg}
+	return &http.Client{Timeout: time.Second * 30, Transport: transport}
+}
```

Comment:
IMHO it makes sense to have both functions here for context, with the original named like `CreateCryptoClient` or smth...
WDYT?